src/pages/Home.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import TextField from '@mui/material/TextField';
+import Autocomplete from '@mui/material/Autocomplete';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import Grid from '@mui/material/Grid';
+import Stack from '@mui/material/Stack';
+import Container from '@mui/material/Container';
+
+import '../styles/Home.scss'
+
+function Home() {
+    return (
+        <Container fixed className="ch-home-page">
+            <Grid container spacing={{ xs: 2, md: 3 }} columns={{ xs: 4, sm: 8, md: 12 }} className="ch-grid-fields">
```

Comment:
There is not needed to add `ch-` prefix to class name. We only add it to class names that are assigned to main container of component